File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/sso/GetUserInfoResVo.java
package org.javamaster.invocationlab.admin.model.sso;

import lombok.Data;

@Data
public class GetUserInfoResVo {
    private String email;
    private String account;
    private String realName;
    private String blood;
    private String mobileNo;
    private String seqnr;
    private Integer accountType;
    private String sex;
    private String empType;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/sso/GetUserInfoResVo.kt
package org.javamaster.invocationlab.admin.model.sso

import org.javamaster.invocationlab.admin.annos.AllOpen


@AllOpen
class GetUserInfoResVo {
    var email: String? = null
    var account: String? = null
    var realName: String? = null
    var blood: String? = null
    var mobileNo: String? = null
    var seqnr: String? = null
    var accountType: Int? = null
    var sex: String? = null
    var empType: String? = null
}
